libs/ngxtension/inject-local-storage/src/inject-local-storage.ts

```diff
@@ -0,0 +1,69 @@
+import { effect, signal, type WritableSignal } from '@angular/core';
+
+export type LocalStorageOptions<T> = {
+	defaultValue?: T | (() => T);
```

Comment:
Can we add some comments for each option here? 

It would make it easier when passing these options and to read some kind of comments on what they do and how they modify the way this function works.

Reply:
I agree, added some comments :)